src/phoenix/server/api/types/Dataset.py

```diff
@@ -59,3 +61,72 @@ async def versions(
             for version in versions
         ]
         return connection_from_list(data=data, args=args)
+
+    @strawberry.field
+    async def examples(
+        self,
+        info: Info[Context, None],
+        dataset_version_id: Optional[GlobalID] = UNSET,
+        first: Optional[int] = 50,
+        last: Optional[int] = UNSET,
+        after: Optional[CursorString] = UNSET,
+        before: Optional[CursorString] = UNSET,
+    ) -> Connection[DatasetExample]:
+        args = ConnectionArgs(
+            first=first,
+            after=after if isinstance(after, CursorString) else None,
+            last=last,
+            before=before if isinstance(before, CursorString) else None,
+        )
+        latest_revisions = select(
+            models.DatasetExampleRevision.dataset_example_id,
+            func.max(models.DatasetExampleRevision.dataset_version_id).label(
+                "latest_dataset_version_id"
+            ),
+        ).group_by(models.DatasetExampleRevision.dataset_example_id)
+        if dataset_version_id:
+            dataset_version_rowid = from_global_id_with_expected_type(
+                global_id=dataset_version_id, expected_type_name="DatasetVersion"
+            )
+            latest_revisions = latest_revisions.where(
+                models.DatasetExampleRevision.dataset_version_id <= dataset_version_rowid
+            )
+        latest_revisions_cte = latest_revisions.cte("latest_revisions")
+        query = (
+            select(models.DatasetExampleRevision, models.DatasetExample.created_at)
+            .join(
+                latest_revisions_cte,
+                onclause=and_(
+                    (
+                        models.DatasetExampleRevision.dataset_example_id
+                        == latest_revisions_cte.c.dataset_example_id
+                    ),
+                    (
+                        models.DatasetExampleRevision.dataset_version_id
+                        == latest_revisions_cte.c.latest_dataset_version_id
```

Comment:
maybe too verbose? haha
```suggestion
                        == latest.c.version_id
```

Reply:
meet you halfway haha

addressed in 57bb281c33dd018010734ec3895648dd4b359371